wasm2c/wasm-rt-impl.c

```diff
@@ -340,6 +340,8 @@ const char* wasm_rt_strerror(wasm_rt_trap_t trap) {
       return "Uncaught exception";
     case WASM_RT_TRAP_UNALIGNED:
       return "Unaligned atomic memory access";
+    case WASM_RT_TRAP_NULL_REF:
+      return "Null reference";
```

Comment:
Is this actually possible?  Doesn't the type system / validation ensure no null refrences can exist at the point of use?   I guess it depends if excref is nullable?

Reply:
yes, exnref can be null, tho the tests never throw a null exnref as far as we can tell.